chpl/chpl-api/e2e/collections/search-controller.postman_collection.json

```diff
@@ -237,6 +257,130 @@
 				"description": "POST /search/beta - filters other than compliance filters - should return 200 status"
 			},
 			"response": []
+		},
+		{
+			"name": "POST - /search/beta - using IS_ELIGIBLE should give 500 status",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							""
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"POST /search/beta request using IS_ELIGIBLE filter should return 500 status and error\", function () {\r",
+							"    pm.response.to.have.status(500);\r",
```

Comment:
IS_ELIGIBLE parameter is being removed and I think the aqa test to check what it does on next line (#279) is good but running a test every day to test that the request with removed parameter returns 500 does not add value. Same comment for test at line #320

Reply:
I would recommend keeping the test for POST but for positive scenario